hw/ip/otbn/data/base-insns.yml

```diff
@@ -448,12 +448,12 @@
     The number of instructions in the loop is given in the `bodysize` immediate.
 
     The `LOOP` instruction doesn't support a zero iteration count.
-    If the value in `grs` is zero, OTBN stops with the `ErrCodeLoop` error.
+    If the value in `grs` is zero, OTBN stops with the `ErrBitLoop` error.
```

Comment:
Can we change this sort of sentence to something more like "... OTBN stops, setting `ErrBitLoop` in `ERROR_BITS`" ? The way it's currently drafted makes it look like we "stop with a particular error", which isn't quite right in a world where there's more than one that can happen at once.